c10/util/Exception.h

```diff
@@ -154,6 +154,12 @@ C10_API void set_warning_handler(WarningHandler* handler) noexcept(true);
 /// Gets the global warning handler.
 C10_API WarningHandler* get_warning_handler() noexcept(true);
 
+/// The TORCH_WARN_ONCE macro is difficult to test for. Use
+/// setWarnOnlyOnce(false) to turn it into TORCH_WARN, which can be
```

Comment:
This comment needs to be updated to reflect the most recent names

Reply:
done